pkg/api/validation/validation_test.go

```diff
@@ -874,6 +911,11 @@ func TestValidatePersistentVolumeClaimUpdate(t *testing.T) {
 			oldClaim:          validUpdateClaim,
 			newClaim:          invalidUpdateClaimAccessModes,
 		},
+		"invalid-update-change-storage-class-annotation-after-creation": {
```

Comment:
@jsafrane 

I updated the tests - added a valid change to annotation and a test to make sure an annotation could be added

```
+++ [1116 12:03:37] Running tests without code coverage
ok  	k8s.io/kubernetes/pkg/api/validation	0.341s
```